R/utility.R

```diff
@@ -166,13 +166,21 @@ waitForNodesToComplete <- function(clusterId, timeout = 86400){
 getJobResult <- function(jobId = "", ...){
   args <- list(...)
 
+  print("Getting job results...")
```

Comment:
cat("Getting job results. . .", fill = TRUE) 

We should be using cat instead of print since print returns an object. There's no reason to return these strings.
I'll be changing all of the other print statements to be cat in my next PR.